### DISCUSSION TITLE
Deprecated?

### DISCUSSION THREAD

**Discussion by HasanYldz** [roman-right/bunnet#26] [Question]
@roman-right Should we still use this fork or is it no longer maintained? I can help to merge beanie 1.26 to here if you still want to keep this library.

**Comment by roman-right** [comment]
Hi! I update Bunnet from Beanie 2-4 times per year (after big releases or when a lot of features accumulated). It is not abandoned. I plan to merge Beanie to Bunnet in September next time.

**Reply by roman-right** [reply]
But any help is appreciated for sure! If you can pick this task up, it would be great!

**Reply by HasanYldz** [reply]
Hey, thanks for your reply. I am a little bit busy for next two weeks after that I can help out with maintenance.
